Frontend/ui-driver/src/Screens/DriverProfileScreen/View.purs

```diff
@@ -1109,6 +1108,16 @@ profileOptionsLayout state push =
             ) (optionList state)
       )
   ]
+  where visibilityCondition optionItem = case optionItem.menuOptions of
+                                        GO_TO_LOCATIONS -> state.props.enableGoto
+                                        DRIVER_BOOKING_OPTIONS -> state.data.config.profile.showBookingOption && not (null state.data.downgradeOptions)
+                                        LIVE_STATS_DASHBOARD -> state.data.config.liveStatsDashboard.showLiveStatsDashboard && state.data.config.liveStatsDashboard.liveStatsDashboardUrl /= ""
+                                        _ -> true
+        disableCondition optionItem = case optionItem.menuOptions of
+                                        DRIVER_BOOKING_OPTIONS -> not state.data.activeRCData.rcStatus
+                                        GO_TO_LOCATIONS -> state.data.goHomeActive || state.props.isRideActive
+                                        _ -> false
+        disabledOptionClickable optionItem = optionItem.menuOptions /= DRIVER_BOOKING_OPTIONS
```

Comment:
This might look much cleaner -
```purescript
where 
    visibilityCondition optionItem = 
      case optionItem.menuOptions of
        GO_TO_LOCATIONS -> state.props.enableGoto
        DRIVER_BOOKING_OPTIONS -> state.data.config.profile.showBookingOption && not (null state.data.downgradeOptions)
        LIVE_STATS_DASHBOARD -> state.data.config.liveStatsDashboard.showLiveStatsDashboard && state.data.config.liveStatsDashboard.liveStatsDashboardUrl /= ""
        _ -> true
        
    disableCondition optionItem = 
      case optionItem.menuOptions of
        DRIVER_BOOKING_OPTIONS -> not state.data.activeRCData.rcStatus
        GO_TO_LOCATIONS -> state.data.goHomeActive || state.props.isRideActive
        _ -> false
    
    disabledOptionClickable optionItem = optionItem.menuOptions /= DRIVER_BOOKING_OPTIONS
```